docs/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@enzymefinance/api-sdk-documentation",
+  "version": "0.0.0",
+  "type": "module",
```

Comment:
this has to be marked as `private: true`. Otherwise `changesets` will publish it on `npm`.

Reply:
We also have to add `@enzymefinance/docs` to the `ignore` list in `.changesets/config`.

Reply:
Done.